CMakeLists.txt

```diff
@@ -199,6 +199,10 @@ option(GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC
        "${GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC_DEFAULT}")
 mark_as_advanced(GOOGLE_CLOUD_CPP_STORAGE_ENABLE_GRPC)
 
+option(GOOGLE_CLOUD_CPP_ENABLE_EXPERIMENTAL_OPENTELEMETRY
```

Comment:
We are moving away from the `GOOGLE_CLOUD_CPP_ENABLE_*` options and trying to consolidate *only* on `GOOGLE_CLOUD_CPP_ENABLE`.  Do you have a reason to support both in this case?

Reply:
I have a weak argument.

1. I assume an option is more discoverable. (I assume there is some way to list all options for a cmake project).
2. Being able to append a `-DGOOGLE_CLOUD_CPP_EXPERIMENTAL_OPENTELEMETRY` adds some amount of convenience to the customers? (vs. appending to `GOOGLE_CLOUD_CPP_ENABLE`)

If you are unconvinced, I'll just close this thing.

Reply:
I am unconvinced.